fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/cascades/predicates/OrPredicate.java

```diff
@@ -291,9 +293,10 @@ private Optional<PredicateMultiMap.PredicateMapping> impliesWithValuesAndRanges(
                              Objects.requireNonNull(foldNullable(Function.identity(),
                                      (queryPredicate, childFunctions) -> queryPredicate.injectCompensationFunctionMaybe(partialMatch,
                                              boundParameterPrefixMap,
-                                             ImmutableList.copyOf(childFunctions)))))));
+                                             ImmutableList.copyOf(childFunctions))))),
+                    Optional.of(this.toResidualPredicate().translateValue(value -> value.translateCorrelations(TranslationMap.rebaseWithAliasMap(aliasMap))))));
```

Comment:
Can you make this `Optional.empty()` as well?

Reply:
done.